controlplane/src/core/composition/composer.ts

```diff
@@ -64,19 +66,40 @@ export class Composer {
    * Build and store the final router config and federated schema to the database. A diff between the
    * previous and current schema is stored as changelog.
    */
-  async deployComposition(composedGraph: ComposedFederatedGraph, composedBy: string) {
+  async deployComposition({
+    composedGraph,
+    composedBy,
+    blobStorage,
+    organizationId,
+  }: {
+    composedGraph: ComposedFederatedGraph;
+    composedBy: string;
+    blobStorage: BlobStorage;
+    organizationId: string;
+  }) {
     const hasCompositionErrors = composedGraph.errors.length > 0;
+    const federatedSchemaVersionId = randomUUID();
 
     let routerConfigJson: JsonValue = null;
+    const path = `${organizationId}/${composedGraph.id}/routerConfigs/latest.json`;
 
     // Build router config when composed schema is valid
     if (!hasCompositionErrors && composedGraph.composedSchema) {
       const routerConfig = buildRouterConfig({
         argumentConfigurations: composedGraph.argumentConfigurations,
         subgraphs: composedGraph.subgraphs,
         federatedSDL: composedGraph.composedSchema,
+        federatedSchemaVersionId,
       });
       routerConfigJson = routerConfig.toJson();
+
+      try {
+        await blobStorage.putObject(path, Buffer.from(JSON.stringify(routerConfigJson), 'utf8'));
```

Comment:
I think you can use `routerConfig.toString()` ?

Reply:
instead of JSON.stringify?

Reply:
yeah, there is not toString, but toJSONString

Reply:
Done